js/main.js

```diff
@@ -31,4 +33,41 @@ Site = {
   },
 };
 
+Site.News = {
+  init: function() {
+    var _this = this;
+
+    _this.shuffleContainer = document.getElementById('shuffle-container');
+
+    _this.bind();
+  },
+
+  bind: function() {
+    var _this = this;
+
+    imagesLoaded(_this.shuffleContainer, function() {
+      _this.initShuffle();
+    });
+  },
+
+  initShuffle: function() {
+    var _this = this;
+
+    $('#shuffle-preloader').remove();
+    $(_this.shuffleContainer).css('opacity', 1)
```

Comment:
http://caniuse.com/#search=opacity coverage for everything other than IE8 so thats fine. imo a problem with the 5 versions autoprefixer, we should probably try to refine that so we can catch those safaris but ignore old IE